speechAssets/PODCAST.txt

```diff
@@ -12,6 +12,6 @@ books podcast
 music weekly
 chips with everything
 politics for humans
-what would a feminist do?
+what would a feminist do
```

Comment:
Again this seems to me really fragile. This information should come from the API and be editable through an editorial tool.